src/configs/sys_config.py

```diff
@@ -0,0 +1,23 @@
+
+# System Configuration
+
+system_config = {
+    "num_users": 4,
+    "experiment_path": "./experiments/",
+    "dset": "cifar10",
+    "dump_dir": "./expt_dump/",
+    "dpath": "/data/unagi0/anakewat/imgs/",
+    "device_ids": {
+        "iid_dispfl": {"node_0": [2], "node_1": [3], "node_2": [4], "node_3": [5], "node_4": [2]},
+        "iid_defkt": {"node_0": [4], "node_1": [4], "node_2": [4], "node_3": [4]},
+        "non_iid": {"node_0": [0], "node_1": [1], "node_2": [2], "node_3": [3]}
+    },
+    "dataset_splits": {
+        "iid": {"num_clients": 4, "samples_per_client": 500},
```

Comment:
no need to mention multiple dataset split configs in a single config. It should looks something like following in my opinion -
```
"dataset_splits": {
  "type": "non-iid",
  "samples_per_user": 500
}
```

Reply:
we should not have `num_clients` here because users is already specified above.